src/attributes.jl

```diff
@@ -2090,64 +2104,169 @@ struct ListOfSupportedNonlinearOperators <: AbstractOptimizerAttribute end
     """
         TerminationStatusCode
 
-    An Enum of possible values for the `TerminationStatus` attribute. This attribute
-    is meant to explain the reason why the optimizer stopped executing in the most
+    An Enum of possible values for the [`TerminationStatus`](@ref) attribute.
+
+    This attribute explains why the optimizer stopped executing in the most
     recent call to [`optimize!`](@ref).
     """,
     TerminationStatusCode,
     "The algorithm has not started.",
     OPTIMIZE_NOT_CALLED,
     # OK
-    "The algorithm found a globally optimal solution.",
+    """
+    The algorithm found a globally optimal solution.
+    """,
     OPTIMAL,
-    "The algorithm concluded that no feasible solution exists.",
+    """
+    The algorithm proved that no primal feasible solution exists.
+    """,
     INFEASIBLE,
-    "The algorithm concluded that no dual bound exists for the problem. If, additionally, a feasible (primal) solution is known to exist, this status typically implies that the problem is unbounded, with some technical exceptions.",
+    """
+    The algorithm concluded that no dual bound exists for the problem.
+
+    If the problem is a conic optimization problem, this status means the dual
+    problem is infeasible..
+
+    If a primal feasible solution exists, this status typically implies that the
+    problem is unbounded, with some technical exceptions (for example, if the
```

Comment:
Not sure why we need strong duality. If we have a both a feasible point and a feasible ray of the primal then we can just start at this feasible point and follow the ray and the objective will be infinite no ?

Reply:
I think the technical exception mentioned before was just about the existence of the feasible point

Reply:
What about a problem like:
```
model = Model()
@variable(model, X[1:2, 1:2], Symmetric)
@constraint(model, X >= 0, set = PSDCone())
@constraint(model, X[1,1] == 0)
@objective(model, Min, 2 * X[1, 2])
```
The primal has a finite optimal primal solution with solution `X = 0`. But the dual is infeasible.